### PR TITLE
Document supported languages

### PR DESCRIPTION
**Pull request by alexplischke**
### Description

Document supported languages (where applicable) for the DevX frameworks.

### CODE REVIEW

**Review by mhan83 — APPROVED** [review]

File: docs/web-apps/automated-testing/cypress.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
 </TabItem>
 </Tabs>
 
+## Supported Languages
+
+Javascript is supported out of the box. TypeScript and Cucumber are also supported, but require additional dependencies at runtime. Please consult our [example repo](https://github.com/saucelabs/saucectl-cypress-example/tree/main/v1/examples) for working end-to-end examples.
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/cypress.md:47]:
```suggestion
JavaScript is supported out of the box. TypeScript and Cucumber are also supported, but require additional dependencies at runtime.  See our [example repo](https://github.com/saucelabs/saucectl-cypress-example/tree/main/v1/examples) for working end-to-end examples.
```

File: docs/web-apps/automated-testing/playwright.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 [Playwright](https://github.com/microsoft/playwright) is a testing framework that you can use to test your web apps &#8212; either locally in Docker or remotely on Sauce Labs Cloud &#8212; using the [`saucectl` CLI](/dev/cli/saucectl). This gives you the flexibility to run your tests in the environment that best suits your organization, while still benefiting from the Sauce Labs vast collection of devices, browser, and operating system combinations and test result data analytics.
 
+## Supported Languages
+
+Javascript and Typescript are supported out of the box.
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/playwright.md:15]:
```suggestion
JavaScript and TypeScript are supported out of the box.
```

File: docs/web-apps/automated-testing/playwright.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+Cucumber.js is not directly supported by playwright. However, playwright can be used as the backing automation framework. Please consult [this setup guide](playwright/cucumber.md) for more information on this topic.
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/playwright.md:16]:
```suggestion
Cucumber.js is not directly supported by Playwright. However, Playwright can be used as the backing automation framework. See [this setup guide](playwright/cucumber.md) for more information.
```

File: docs/web-apps/automated-testing/testcafe.md [hunk 4]
```diff
@@ -42,6 +42,10 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
 </TabItem>
 </Tabs>
 
+## Supported Languages
+
+Javascript and Typescript are supported out of the box.
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/testcafe.md:47]:
```suggestion
JavaScript and TypeScript are supported out of the box.
```

**Comment by alexplischke** [comment]
@aliciakrebs Need re-approval 🙏

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**